packages/remix/package.json

```diff
@@ -21,12 +21,12 @@
     "@sentry/node": "7.1.1",
     "@sentry/react": "7.1.1",
     "@sentry/tracing": "7.1.1",
+    "@sentry/types": "7.1.1",
     "@sentry/utils": "7.1.1",
     "@sentry/webpack-plugin": "1.18.9",
     "tslib": "^1.9.3"
   },
   "devDependencies": {
-    "@sentry/types": "7.1.1",
```

Comment:
Why did this get moved?

Reply:
Because it's supposed to be a regular dependency, as it is in all of our other packages. Because we don't ship types directly in our SDKs, our users need `@sentry/types` if they're using TS.